modules/apps/object/object-test/src/testFunctional/tests/ObjectTranslations.testcase

```diff
@@ -156,4 +156,66 @@ definition {
 		}
 	}
 
+	@description = "LPS-198382 - Verify an object entry can contain translations and be restricted by unique values"
+	@priority = 4
+	test CanUseUniqueValue {
+		property custom.properties = "feature.flag.LPS-146755=true";
```

Comment:
@JhossephG should we also remove this one?

Reply:
@JhossephG we should remove this property since the feature flag was removed